<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Цена доплат</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_many_year"/>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="le_one_year"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>За пол года</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>За год</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_ok()</slot>
 </slots>
</ui>
